Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -78,8 +78,10 @@ public extension UIConfig {
 public extension UIConfig {
     struct MessageListUI {
         public var collectionView: ChatChannelCollectionView.Type = ChatChannelCollectionView.self
-        public var collectionLayout: UICollectionViewLayout.Type = ChatChannelCollectionViewLayout.self
+        public var collectionLayout: ChatChannelCollectionViewLayout.Type = ChatChannelCollectionViewLayout.self
         public var minTimeInvteralBetweenMessagesInGroup: TimeInterval = 10
+        /// Point from top, after which we should start loading next batch on messages
+        public var nextMessageLoadingThreshold: CGFloat = 100
```

Comment:
It took me a while to understand what exactly this is. I don't have a better name though :/

Reply:
What about:
```
/// Vertical contentOffset for message list, when next message batch should be requested
public var offsetToPreloadMoreMessages: CGFloat = 100
```